Riot/Modules/Settings/SettingsViewController.m

```diff
@@ -4046,6 +4090,39 @@ - (void)deactivateAccountViewControllerDidCancel:(DeactivateAccountViewControlle
     [deactivateAccountViewController dismissViewControllerAnimated:YES completion:nil];
 }
 
+#pragma mark - NotificationSettingsCoordinatorBridgePresenter
+
+- (void)showNotificationSettings: (NotificationSettingsScreen) screen
+{
+    
+    if (@available(iOS 14.0, *)) {
+        NotificationSettingsCoordinatorBridgePresenter *notificationSettingsBridgePresenter = [[NotificationSettingsCoordinatorBridgePresenter alloc] initWithSession:self.mainSession];
+        notificationSettingsBridgePresenter.delegate = self;
+        
+        MXWeakify(self);
+        
+        [notificationSettingsBridgePresenter pushFrom:self.navigationController animated:YES screen:screen popCompletion:^{
+            MXStrongifyAndReturnIfNil(self);
+            
+            self.notificationSettingsBridgePresenter = nil;
+        }];
+        
+        
+        self.notificationSettingsBridgePresenter = notificationSettingsBridgePresenter;
+    } else {
+        // Fallback on earlier versions
```

Comment:
Is there a (planned) fallback for earlier versions?

Reply:
Updated in a subsequent commit, we do the version checks further up in the file and then remove this else condition.